src/Sql/Sql.Sdk/Generated/Models/ServerBlobAuditingPolicy.cs

```diff
@@ -112,7 +112,7 @@ public ServerBlobAuditingPolicy()
         /// audit logs.
         ///
         /// For more information, see [Database-Level Audit Action
-        /// Groups](https://learn.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups).
+        /// Groups](https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups).
```

Comment:
The docs URL has been updated to learn.microsoft.com instead of docs.microsoft.com. Please revert the change and re-generate the code.

Reply:
My commit was reverted in this [commit](https://github.com/Azure/azure-powershell/commit/ec9684cc05ff91bab74e3f4e9b699415877f8c4a). From here it started the breakage in the build